### DISCUSSION TITLE
How to create sections without creating a title slide

### DISCUSSION THREAD

**Discussion by mathias-aparicio** [touying-typ/touying#138] [Q&A] [answered]
I am very pleased wit touying and the university theme, 
I want to create sections so that the section appears in the footer like Introduction, part 1, part 2, conclusion. But I don't want to create a slides in the same way with an underline title.
I create the section with `=`

Thanks in advance for your answer.

**Comment by Tinggaard** [comment]
Yes, you should be able to achieve that using the `<touying:hidden>` label.

```typ
= Hidden Title <touying:hidden>

== Slide

Content goes here as per usual.
```

**Comment by OrangeX4** [comment, marked as answer]
config-common(new-section-slide-fn: none)

**Comment by mathias-aparicio** [comment]
Thank you for your answer it works as intended with 
```typst
#show: university-theme.with(
  config-common(new-section-slide-fn: none),
)
```

and without `touying-hidden`
